extensions/adobe/experience/customerJourneyManagement/messageprofile.example.1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+  "xdm:channel": {
```

Comment:
shouldn't this be an array. An profile can be associated with multiple channels ?

Reply:
But for each channel we would be ingesting the separate EE.